app/views/shared/vacancy/_share_buttons.html.slim

```diff
@@ -1,11 +1,11 @@
 ul.govuk-list
   li.share-links__list-item
-    a target="_blank" class="govuk-link share-links__link vacancy-share-link" href="https://www.facebook.com/sharer/sharer.php?u=#{CGI.escape(@vacancy.share_url)}" data-target="facebook"
+    a target="_blank" class="govuk-link share-links__link vacancy-share-link" href=new_share_path(channel: "facebook", vacancy_id: @vacancy.id) data-target="facebook"
       img src=asset_pack_path("media/images/svg/logo-facebook.svg") height="40" width="40" class="share-links__link-icon" aria-hidden="true"
       span.visually-hidden = t("jobs.share_on")
       | Facebook
   li.share-links__list-item
-    a target="_blank" class="govuk-link share-links__link vacancy-share-link" href="https://twitter.com/share?url=#{CGI.escape(@vacancy.share_url)}&text=#{CGI.escape(@vacancy.job_title_and_parent_organisation_name)}" data-target="twitter"
+    a target="_blank" class="govuk-link share-links__link vacancy-share-link" href=new_share_path(channel: "twitter", vacancy_id: @vacancy.id) data-target="twitter"
```

Comment:
Can we make this a `link_to` so it's more obvious it goes to a route (as opposed to a generic `<a>`)?